stripe_donations/controllers.py

```diff
@@ -1230,9 +1229,8 @@ def local_datetime_string_from_utc_timestamp(utc_timestamp):
     offset = datetime.fromtimestamp(now_timestamp) - datetime.utcfromtimestamp(now_timestamp)
     utc_datetime = datetime.utcfromtimestamp(utc_timestamp)
     naive_local_datetime = utc_datetime + offset
-    # time_zone = pytz.timezone("America/Los_Angeles")
-    # local_datetime = time_zone.localize(naive_local_datetime)
-    local_datetime = get_timezone_and_datetime_now(datetime_obj=naive_local_datetime)[1]
+    time_zone = pytz.timezone("America/Los_Angeles")
+    local_datetime = time_zone.localize(naive_local_datetime)
```

Comment:
Did you mean to roll these changes back?

Reply:
I think again, because this logic is only found here, I decided to not replace it with the function I created to replace it. Hope that makes sense.